plot_altair.py

```diff
@@ -10,11 +10,6 @@
 import larq as lq
 import pandas as pd
 
-try:
-    tf.enable_eager_execution()
-except Exception:
-    pass
```

Comment:
Why was this previously necessary?

Reply:
For TF 1.x as this script used to be in larq/larq